maquetado/scss/styles.scss

```diff
@@ -1,32 +1,56 @@
 .bold {
-    font-weight: bold;
+  font-weight: bold;
+}
+
+.book-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+}
+
+.book-cover {
+  width: 261px;
+  height: 368px;
+}
+
+.column {
+  display: flex;
+  flex-direction: column;
+  width: 400px;
```

Comment:
I wouldn't fix this width. You can set `flex-grow: 1` so it grows all it can grow inside the "row"

Also, if you are going to make this `column` class, it might be better to make it more "generic" in case you eventually want to have another "column". I would remove this and leave only the display and flex-direction, and I'd make another class called `book-information-container` or something that has the specific properties this div needs, then add both this classes to the div.

Reply:
The same applies to classes like `.book-container`, you can use `row` or `column` in the html instead of setting `display: flex` inside it. For that type of things it's better to use more generic classes and leave these for more specific things like `font-size`, `position`, `etc`.